dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipOutboundConnLink.java

```diff
@@ -59,27 +66,33 @@ public abstract class SipOutboundConnLink extends SipConnLink {
 	 * @param channel  channel that created this connection
 	 */
 	public SipOutboundConnLink(String peerHost, int peerPort, SipInboundChannel sipInboundChannel, Channel channel, boolean isSecure) {
-        super(peerHost, peerPort, sipInboundChannel, channel);
-        m_chainName = sipInboundChannel.getOutboundChainName();
-        bootstrap.group(workerGroup).channel(NioSocketChannel.class);
-    
-        bootstrap.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, s_connectTimeout);
-        bootstrap.handler(new ChannelInitializer<SocketChannel>() {
-            @Override
-            public void initChannel(SocketChannel ch) throws Exception {
-                final ChannelPipeline pipeline = ch.pipeline();
-                if (isSecure) {
-                    SslContext context = GenericEndpointImpl.getTlsProvider().getOutboundSSLContext(GenericEndpointImpl.getSslOptions(), peerHost, Integer.toString(peerPort));
-                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                        Tr.debug(this, tc, "SipOutboundConnLink", "context: " + context);
-                    }
-                    SSLEngine engine = context.newEngine(ch.alloc());
-                    pipeline.addFirst("ssl", new SslHandler(engine, false));
-                }
-                pipeline.addLast("decoder", new SipMessageBufferStreamDecoder());
-                pipeline.addLast("handler", new SipStreamHandler());
-            }
-        });
+          super(peerHost, peerPort, sipInboundChannel, channel);
+          m_chainName = sipInboundChannel.getOutboundChainName();
+          Map<String, Object> tcpOpt = new HashMap<>();
+          nettyFw = GenericEndpointImpl.getNettyBundle();
+          try {
+            bootstrap = nettyFw.createTCPBootstrapOutbound(tcpOpt); 
```

Comment:
Could also consider using `Collections.emptyMap()` for this instead of just the HashMap constructor since we technically just pass in an empty map with no operations done to it